internal/moovdata/moov.go

```diff
@@ -12,9 +13,12 @@ var (
 	// mvhdBoxPath is the BoxPath to the MVHD Box that contains Creation Time metadata
 	// Ref: https://developer.apple.com/documentation/quicktime-file-format/movie_header_atom/creation_time
 	mvhdBoxPath mp4.BoxPath = []mp4.BoxType{mp4.BoxTypeMoov(), mp4.BoxTypeMvhd()}
-	// metaBoxPath is the BoxPath to the Meta Box the contains tags that may contain a CreationDate tag key
-	// Ref: https://developer.apple.com/documentation/quicktime-file-format/metadata_atom
-	//metaBoxPath mp4.BoxPath = []mp4.BoxType{mp4.BoxTypeMeta()}
+	// keysBoxPath is the BoxPath to the Keys Box. A keys box's type should
+	// be intepreted before any other fields in Meta box so we can identifiy a proper index for items in ilst
+	// Ref: https://developer.apple.com/documentation/quicktime-file-format/metadata_handler_atom
+	keysBoxPath mp4.BoxPath = []mp4.BoxType{mp4.BoxTypeMoov(), mp4.BoxTypeMeta(), mp4.BoxTypeKeys()}
+	// ilstBoxAnyPath is the BoxPath to an AnyBoxI under ilst.
+	ilstBoxAnyPath mp4.BoxPath = []mp4.BoxType{mp4.BoxTypeMoov(), mp4.BoxTypeMeta(), mp4.BoxTypeIlst(), mp4.BoxTypeAny()}
```

Comment:
This path with `BoxTypeAny` fails to pickup the new `Item` BoxType.. Need to circle back to figure out the appropriate path to pickup the new `Item` Block type I added in the above PR. 


```
box info not found
```